### ISSUE TITLE
Filters keys no longer have expected values

### ISSUE THREAD

**Issue by kkappler** [kujaku11/mt_metadata#58]
We had talked about adding the station label as part of the filter key, but it looks like the string representation of the station object wound up getting associated with the filter key, not the label.  

i.e. instead of prepending 

"PKD"

this was prepended instead:

```
"station pkd (bear valley ranch, parkfield, ca, usa)\n\tstation code: pkd\n\tchannel count: 4 per 373 (selected per total)\n\t1996-09-06t16:28:00.000000z - \n\taccess: open \n\tlatitude: 35.95, longitude: -120.54, elevation: 583.0 m\n\tavailable channels:\n\t\tpkd..bf1, pkd..bf2, pkd..bq1, pkd..bq2_bq1_0'
```

![Screenshot from 2021-09-14 16-26-48](https://user-images.githubusercontent.com/1154939/133347050-2341b630-5abe-4059-886d-786bbdc05a64.png)

**Comment by kkappler** [comment]
Assigning to myself -- I think this problem actually lives in aurora -- will update ASAP

**Comment by kkappler** [comment]
My bad, this was an aurora issue.  Sorry!  Fixed with aurora commit 56748b4e5555f96ce75c42048744b7156c083da1
